core/modules/node/css/node.preview.css

```diff
@@ -0,0 +1,98 @@
+
+/**
+ * @file
+ * Styles for preview pages.
+ */
+
+.node-preview-container {
+  font: normal 16px/1.538 "Open Sans", "Lucida Grande", "Lucida Sans Unicode", sans-serif;
+  background-color: #efefef;
+  z-index: 999;
+  width: 100%;
+  padding: 10px;
+  overflow: auto;
+}
+.page.preview-js-processed .node-preview-container {
```

Comment:
This isn't called on the test box. Maybe use `html.js .node-preview-container`?

Reply:
This works as expected. `.page.preview-js-processed` only exists if JS is on.

Reply:
I don't understand. On the text box the ribbon is showing up on the top for me. JS is enabled but the CSS should be putting it on the bottom I assume.

Reply:
Browser issue maybe? On Chrome and Edge its on the bottom for me.

![prev](https://user-images.githubusercontent.com/1042283/44698428-78be4180-aa45-11e8-8ccd-4b743ef40799.PNG)